zk/stages/stage_sequence_execute.go

```diff
@@ -419,6 +423,13 @@ func sequencingBatchStep(
 				default:
 				}
 
+				sender, ok := transaction.GetSender()
```

Comment:
@hexoscott we ll have a bit of a merge conflict with zkevm, though easy to resolve, as I already fixed getting sender there. And we break the loop early if we failed to restrieve the sender, so the change below wont be needed.
I think we should unify it somehow with the zkevm branch. WDYT?